src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -253,6 +253,10 @@ var dynamicTable = new Class({
                     if ((state != 'downloading') && !~state.indexOf('DL'))
                         return false;
                     break;
+                case 'seeding':
+                    if ((state != 'uploading') && (state != 'stalledUP') && (state != 'queuedUP'))
```

Comment:
But in `TorrentModel::getTorrentStatusReport()` it gets registered as `seeding` though...